tests/test_report.py

```diff
@@ -0,0 +1,132 @@
+import pytest
```

Comment:
The tests in this file are rather hard to maintain, as you might have seen when adding the extra space before the `#`. I think we have to think about what we actually want to test here. Having one or two tests that check the output is fine, but we already have other tests for checking the sorting and the filtering by minimum confidence.

On the other hand, some tests are missing. For example, I don't think the `--make-whitelist` feature works for properties or attributes at the moment. We should check that `vulture --make-whitelist myfile.py > whitelist.py` results in `vulture whitelist.py myfile.py` exiting with 0.

Reply:
Yes, indeed the previous version of tests was very verbose and fragile - I've only very basic tests for ensuring that the output format remains intact. Also, checking for the exit code was a great idea - I've tried implementing a version of it in `tests/test_make_whitelist.py`. I've also included the missing tests for attributes and properties. Let me know if it could be improved.